src/query/vacuum.c

```diff
@@ -1124,27 +1125,29 @@ xvacuum_dump (THREAD_ENTRY * thread_p, FILE * outfp)
 
   fprintf (outfp, "\n");
   fprintf (outfp, "*** Vacuum Dump ***\n");
-  fprintf (outfp, "First log page ID referenced = %lld ", min_log_pageid);
-
-  if (logpb_is_page_in_archive (min_log_pageid))
+  fprintf (outfp, "First log page ID referenced = %lld", (long long int) min_log_pageid);
```

Comment:
are we still using fprintf? the previous fprintf's can also be switched to ofstream...

Reply:
given it is existing code, I preferred not to touch it too much;
I did the `long long int` explicit cast to remove a compilation warning; I tried to use the `PRId64` macro but, for a reason I did not understand, CircleCI compilation failed - https://circleci.com/gh/CUBRID/cubrid/24547?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link - (although the macro is used in few other areas of the code)